packages/forms/resources/views/components/toggle.blade.php

```diff
@@ -19,8 +37,8 @@
                 x-bind:aria-checked="state.toString()"
                 x-on:click="state = ! state"
                 x-bind:class="{
-                    'bg-primary-600': state,
-                    'bg-gray-200 @if (config('forms.dark_mode')) dark:bg-white/10 @endif': ! state,
+                    '{{$onColorClass}}': state,
```

Comment:
Please add the match statements inside these `{{ match(...) }}` instead of adding extra variables